packages/build/src/build/buildPages.js

```diff
@@ -30,8 +30,45 @@ Blocks:
   - set request ids
   - set block meta
   - set blocks to areas.content
+  - set operators list on context blocks
 */
 
+function getContextOperators(block) {
+  const stripContext = (_, value) => {
+    if (get(value, 'meta.category') === 'context') {
+      return null;
+    }
+    return value;
+  };
+  const { requests, ...webBlock } = block;
+  webBlock.areas = JSON.parse(JSON.stringify(webBlock.areas || {}), stripContext);
+  const operators = new Set();
+  const pushOperators = (_, value) => {
+    if (type.isObject(value) && Object.keys(value).length === 1) {
+      const key = Object.keys(value)[0];
+      const [op, _] = key.split('.');
+      const operator = op.replace(/^(\_+)/gm, '_');
```

Comment:
maybe just `if (op.startsWith('_'))`?
Why do we need to regex replace here?

Reply:
Because it replaces all multiples of '_'  from the start with a single '_'.